riscv/csrs.h

```diff
@@ -675,7 +680,7 @@ class dcsr_csr_t: public csr_t {
   dcsr_csr_t(processor_t* const proc, const reg_t addr);
   virtual void verify_permissions(insn_t insn, bool write) const override;
   virtual reg_t read() const noexcept override;
-  void write_cause_and_prv(uint8_t cause, reg_t prv, bool v) noexcept;
+  void update_fields(uint8_t cause, reg_t prv, bool v, const elp_t pelp) noexcept;
```

Comment:
Be consistent about adding `const` here. In the definition, you added `const` on all 4 params. (It's not necessary to add `const` here at all, but if you want to add it, at least add it to all 4.)

Reply:
Yeah, I think you're right. I find out that I did add `const` to these parameters in the method definition. It's weird that the program got linked successfully.